heed/src/mdb/lmdb_error.rs

```diff
@@ -53,6 +53,13 @@ pub enum Error {
     /// Transaction cannot recover - it must be aborted.
     BadTxn,
     /// Unsupported size of key/DB name/data, or wrong DUP_FIXED size.
+    ///
+    /// Common causes of this error:
+    ///   - You tried to store a key longer than the max allowed key (511 bytes by default)
```

Comment:
```suggestion
    ///   - You tried to store a zero-length key
```

Reply:
I added the additional "You tried to store a zero-length key" bullet point. I'm not sure you meant to remove the "You tried to store a key longer than the max allowed key (511 bytes by default)" one.

Reply:
Whoops, it's a mistake. Sorry, I forgot to copy/paste/edit, I just edited the line 😄